xbmc/windowing/windows/WinSystemWin32.h

```diff
@@ -153,6 +153,7 @@ class CWinSystemWin32 : public CWinSystemBase
   virtual bool Restore();
   virtual bool Hide();
   virtual bool Show(bool raise = true);
+  virtual void Resize();
```

Comment:
XBMC_VIDEORESIZE is not handled in true fullscreen mode see https://github.com/xbmc/xbmc/blob/master/xbmc/Application.cpp#L340